lib/remap-functions/Cargo.toml

```
@@ -128,6 +129,7 @@ parse_grok = ["grok"]
 parse_json = ["serde_json"]
 parse_regex = ["regex"]
 parse_regex_all = ["regex"]
+parse_key_value = []
```

```suggestion
parse_key_value = ["nom"]
```

(you can validate this by running `cargo build --no-default-features --features parse_key_value`)

---

:thinking: I wonder if we should add this sort of check to CI like we have for `make check-components`.